src/index.js

```diff
@@ -0,0 +1,223 @@
+const state = {
+  currentTemp: 0,
+  cityName: "Miami",
+  defaultCityName: "Seattle",
+};
```

Comment:
Nice work on selecting what pieces of data should go in state! I think that these a the quintessential choices! ✨